channels/ao.m3u

```diff
@@ -2,4 +2,4 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/12aPxO7.png" group-title="General",TPA1
 https://api.new.livestream.com/accounts/1181452/events/8865379/live.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Portuguese" tvg-logo="https://yt3.ggpht.com/ytc/AAUvwng5WHhHdPIDydhUd-TmZXIxqvYqzTmPtCdDtDnmCw=s900-c-k-c0x00ffffff-no-rj" group-title="General",TV Zimbo
-https://api.new.livestream.com/accounts/29819666/events/9254936/live.m3u8
+https://api.new.livestream.com/accounts/29819666/events/9254936/broadcasts/215608865.secure.m3u8
```

Comment:
Link is broken

Reply:
Outdated version of my PR + it tends to break, maybe a timestamped stream.